resources/views/components/ticket/ticket-list.blade.php

```diff
@@ -74,7 +74,7 @@
                                     :consoleName="$game->system->Name"
                                 />
                             </td>
-                            <td>{!! userAvatar($ticket->achievement->author) !!}</td>
+                            <td>{!! userAvatar($ticket->achievement->developer->display_name) !!}</td>
```

Comment:
`userAvatar` accepts a `User`. Suggest just passing `$ticket->achievement->developer` to eliminate re-fetching all the developer UserAccount records.

Reply:
Good point - that slipped my mind. Addressed in latest.